jenkins/getPlatform.py

```diff
@@ -58,7 +58,7 @@
      else:
         compiler = 'gcc' + mobj.group(1) + mobj.group(2)
   elif ccommand.endswith('clang'):
-     versioninfo = os.popen4(ccommand + ' -v')[1].read()
+     versioninfo = os.popen(ccommand + ' --version').read()
```

Comment:
Here "-v" goes to `stderr`, but `--version` goes to stdout, which is easier to read portably with popen().